assets/src/edit-story/components/inspector/prepublish/checklistTab.js

```diff
@@ -87,11 +87,12 @@ const Row = styled.button`
     outline: 2px solid ${({ theme }) => theme.colors.accent.primary};
     outline-offset: 5px;
   }
-  ${({ onClick }) =>
-    Boolean(onClick) &&
-    `&:hover {
-      cursor: pointer;
-    }`}
+`;
+
+const Underline = styled.u`
+  &:hover {
+    cursor: pointer;
+  }
```

Comment:
can we do `span` and add `text-decoration: underline` instead?

Reply:
Also should just be able to put `cursor: pointer` always and not just on hover.